src/System.Private.ServiceModel/src/project.json

```diff
@@ -50,12 +50,14 @@
   },
```

Comment:
You can if you like but it isn't strictly required.  I was primarily trying to do the smallest change to unblock.

What I'd like to see us do is use the latest packages for all "netstandard" sections and let the version of netstandard determine the right references rather than all of this cherry picking.